core/src/main/java/org/apache/carbondata/core/datastore/page/encoding/adaptive/AdaptiveIntegralCodec.java

```diff
@@ -23,6 +23,7 @@
 import java.util.BitSet;
 import java.util.List;
 import java.util.Map;
+import java.util.Stack;
```

Comment:
You have missed to handle adaptive **delta** flows
AdaptiveDeltaIntegralCodec
AdaptiveDeltaFloatingCodec

Reply:
okay